### DISCUSSION TITLE
Audio with no speech return a wrong transcript

### DISCUSSION THREAD

**Discussion by arifqieyah** [deepgram/community#388] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I recorded an audio but with no speech at all, my expectation Deepgram should return the transcript as empty string but the transcript return "je ne sais pas s'il vous plaît" I noticed this is only happens when I use language=fr and model=enhanced

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?model=enhanced&language=fr

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

f52edcf2-caee-46cf-9996-239ae9f8feeb

### If possible, please attach an example audio file to reproduce the issue.

https://github.com/deepgram/community/assets/13045449/4fbdb464-5c30-4a33-b768-1c15c70e48f1

**Comment by nikolawhallon** [comment, marked as answer]
Some models can hallucinate, we have seen this with, for example, audio files that contain true silence (as opposed to background noise). We have an algorithm in the works which solves this particular issue, I can test if it will solve your particular issue too (seems likely)!
